docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,107 @@
+=====================
+RPM rsync Distributor
+=====================
+
+Purpose:
+========
+The RPM rsync distributor publishes RPM content to a remote server. As the name suggests, the
+distributor uses rsync to copy.
+
+Configuration
+=============
+Here's example of rpm_rsync_distributor configuration:
+
+.. code-block:: json
+
+    {
+     "distributor_id": "rpm_rsync_distributor",
+     "distributor_type_id": "rpm_rsync_distributor",
+     "distributor_config": {
+        "remote": {
+            "auth_type": "publickey",
+            "ssh_login": "foo",
+            "key_path": "/home/user/.ssh/id_rsa",
+            "host": "192.168.121.1",
+            "root": "/home/foo/pulp_root_dir"
+        },
+        "predistributor_id": "yum_distributor",
+        "force_full": true
+    },
+     "auto_publish": false
+    }
+
+``auth_type``
+  Two authentication methods are supported: 'publickey' and 'password'.
+
+``ssh_login``
+  ssh login to remote machine
+
+``key_path``
+  A path to the private key that will be used as identity file for ssh. When ``auth_type`` is
+  'publickey' this is a required config.
```

Comment:
This should also note that the key must be readable by user `apache`. I had my key owned by a different user and it couldn't read it.